### DISCUSSION TITLE
example/axum_example  can't build

### DISCUSSION THREAD

**Discussion by jellybobbin** [SeaQL/sea-orm#905] [Q&A] [answered]
axum_example can not build  when i delete  line 28 in Cargo.toml (path = "../../" # remove this line in your own project)

**Comment by billy1624** [comment, marked as answer]
Hey @jellybobbin, thanks for asking!! It's expected and normal. You need to remove `path = "..."` on three `Cargo.toml` at the same time:
- https://github.com/SeaQL/sea-orm/blob/41116499e0ee27ae20cc5ae6295b83d9f910dce4/examples/axum_example/Cargo.toml#L27-L36
- https://github.com/SeaQL/sea-orm/blob/41116499e0ee27ae20cc5ae6295b83d9f910dce4/examples/axum_example/migration/Cargo.toml#L14-L21
- https://github.com/SeaQL/sea-orm/blob/41116499e0ee27ae20cc5ae6295b83d9f910dce4/examples/axum_example/entity/Cargo.toml#L14-L16

**Comment by jellybobbin** [comment]
@billy1624  In fact, I did it before I asked。but now it's ok. thx

**Comment by billy1624** [comment]
Welcomed!
